ui/app/components/app/metamask-template-renderer/metamask-template-renderer.js

```diff
@@ -0,0 +1,64 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import { safeComponentList } from './safe-component-list'
+
+const MetaMaskTemplateRenderer = ({ sections }) => {
+  if (!sections) {
+    return null
+  }
+  if (typeof sections === 'string') {
+    return sections
+  }
+  return (
+    <>
+      {sections.reduce((acc, child, index) => {
```

Comment:
nit: We should rename `acc` to `allChildren` (or something better) for great specificity while avoiding a collision with the `children` constant assigned on L21.

Reply:
good call :D